go/pkg/bertyprotocol/contact_request_manager.go

```diff
@@ -267,7 +266,7 @@ const contactRequestV1 = "/berty/contact_req/1.0.0"
 
 func (c *contactRequestsManager) incomingHandler(stream network.Stream) {
 	defer func() {
-		if err := p2phelpers.FullClose(stream); err != nil {
```

Comment:
I would definitely just call `stream.Close` here as there's nothing you can do if that fails.